### PR TITLE
changing titles

### PR DESCRIPTION
**Pull request by Marchela04**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution.

To ensure that someone gets notified, you can also click on "Request a review" and add
the suggested person.

Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact the HSF training convenors.

You may delete these instructions from your comment.

\- HSF Training
</details>

### CODE REVIEW

File: _episodes/02-Basics_and_exploring_tuples.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Physics background"
+title: "Basic and exploring our tuples"
```
Comment (klieret) [inline _episodes/02-Basics_and_exploring_tuples.md]: Basics? Or perhaps "Basic data exploration"?

File: _episodes/03-More_advance_plots.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Data Discussion"
+title: "More advance plots"
```
Comment (klieret) [inline _episodes/03-More_advance_plots.md]: advanced

File: _episodes/02-Basics_and_exploring_tuples.md (hunk 1 above)
Comment (Marchela04) [reply _episodes/02-Basics_and_exploring_tuples.md]: Thanks, we will start pushing changes to start to have some progress and be able to see the progress of the other members, this is not going to be the final name, but is more close.

File: _episodes/03-More_advance_plots.md (hunk 2 above)
Comment (Marchela04) [reply _episodes/03-More_advance_plots.md]: I will correct this sorry

**Comment by michmx** [comment]
Seems the `.idea` directory from your IDE was pushed in one of the commits. Could you remove it in a next commit?

**Pull request closed without merging**